configs/default/FLMD-GRAVITYF7.config

```diff
@@ -0,0 +1,107 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.5 Mar 16 2020 / 05:21:46 (d4e74e39c) MSP API: 1.42
+
+board_name GRAVITYF7
+manufacturer_id FLMD
```

Comment:
This manufacturer id does not exist.

Reply:
https://github.com/betaflight/unified-targets/issues/236